### DISCUSSION TITLE
CSLA version 7.0.4 release

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#3795] [Announcements]
I am pleased to announce that CSLA 7 version 7.0.4 has been released and is available in NuGet.

This is a bug fix release over 7.0.3.

https://github.com/MarimerLLC/csla/releases/tag/v7.0.4

Thanks to @StefanOssendorf for helping to get this release built and available.

**Comment by Chicagoan2016** [comment]
@rockfordlhotka  I wonder why do we have a release for previous version? (current stable version being 8.0.1).
I am assuming the current stable version doesn't need (the aforementioned) bug fixes.
Kind Regards

**Reply by rockfordlhotka** [reply]
Generally I am willing to accept bug fix PRs for the previous primary version. If someone has an issue they're willing to backport a fix for into the version they are using (just one back) I'm happy to support that.
